docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -93,3 +93,12 @@ removed based on age.
 The minimal age of a query in the history before it is expired. An expired
 query is removed from the query history buffer and no longer available in
 the :doc:`/admin/web-interface`.
+
+``query.max-cpu-time``
+^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-duration`
+* **Default value:** ``1_000_000_000d``
+
+Limits CPU time used by a query. If a limit is not set, CPU usage can exceed
+processing limits.
```

Comment:
```suggestion
This is the max amount of CPU time that a query can use across the entire cluster. Queries that exceed this limit will be killed [with an EXCEEDED_CPU_LIMIT error/INSUFFICIENT_RESOURCES error].
```

The part in `[]` is optional - I'd recommend not including it.

However the bigger part is that "Memory management" page should be part of "Query management". Or the memory management page should be renamed to "Resource management" and this new config should be documented under there.

Reply:
@hashhar Thank you. This is a mistake commit resulting from an erroneous squashing process.  The original copy read: Limits CPU time used by a query. Queries that use more CPU time are cancelled. 

As far as your suggestion to add this property to the memory management page, would it belong there, given it's a duration type property?

Reply:
I mean that the Memory management page should be renamed to "Resource management" and this config can then be documented there since this config also deals with limiting resources like the ones under "Memory management".